Makefile

```diff
@@ -36,7 +36,7 @@ BUILD_DATE=$$(date --utc -Iseconds)
 export GO111MODULE=on
 
 .PHONY: build
-build: gofmt golint govet dist
+build: gofmt golint govet dist generate-manifests-tree
```

Comment:
why are we adding this to build ?

Reply:
because this is the target the openshift CI calls. I believe this is OK because we just create different manifest tree that only the new targets cares about (metadata-container related targets), but I'm open to suggestions here.